gulpfile.js

```diff
@@ -112,7 +112,9 @@ function createWebpackConfig(defines, output) {
       loaders: [
         {
           loader: 'babel-loader',
+          exclude: /src\/core\/(glyphlist|unicode)/, // babel is too slow
           options: {
+            presets: ['es2015'],
```

Comment:
Unless I'm completely mistaken, this will cause all build targets to output ES5 compatible code. For e.g. `GENERIC` builds that makes sense, but for e.g. `FIREFOX` or `MOZCENTRAL` I think it'd be nice if we would be able to keep the code as ES6.

Could we perhaps utilize the `PDFJS_NEXT` build flag, and only add this `es2015` preset when `PDFJS_NEXT === true`?